copy.html

```diff
@@ -0,0 +1,239 @@
+  height: 100px;
+}
+
+.grid-item img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.hover-container {
+  display: none;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: white;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+}
+
+.grid-item:hover .hover-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+<div class="blog-containers">
+  <div class="blog">1</div>
+  <div class="blog">2</div> 
+  <div class="blog">3</div>
+  <div class="blog">4</div>
+</div>
+<div class="blog-info">
+  <div class="blog-1">
+      <br>
+      <h5>Breaking Your Coder's<br> Block</h5>
+      <br>
+      <p class="t3">APRIL 29,2016</p>
+      <div class="blog-bar"></div>
+      <p class="t4">At one time or another, as a <br>developer, we've all found <br>ourselves hopelessly stuck<br> on a coding issue....</p>
+  </div>
+  <div class="blog-2">
+      <h5>!Awake: This is Your <br> Brain on Caffeine</h5>
+      <br>
+      <p class="t3">APRIL 29, 2016</p>
+      <div class="blog-bar"></div>
+      <p class="t4">If there's one thing I can truly  <br>claim to be an expert on, it's<br> staying awake. For five years<br> I struggled to stay employed...</p>
+  </div>
+  <div class="blog-3">
+      <h5>Getting a Handle on <br>Handlebars</h5>
+      <br>
+      <p class="t3">MARCH 04, 2015</p>
+      <div class="blog-bar"></div>
+      <p class="t4">Here at EyeCue Lab we <br>render most of our data-laden <br>HTML pages in Handlebar <br>templates...</p>
+  </div>
+  <div class="blog-4">
+      <h5>5 Extensions for Your <br>Chrome Toolbelt</h5>
+      <br>
+      <p class="t3"> FEBRUARY 23, 2015</p>
+      <div class="blog-bar"></div>
+      <p class="t4">If you're not using Google <br>Chrome as a front-end web <br>developer, you're missing out.<br> Not only is Chrome the....</p>
+  </div>
+</div>
+
+
+
+
+.blog-containers{
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  column-gap: 80px;
+  height: 190px;
+  width: 1100px;
+  margin-left: 150px;
+  margin-top: 120px;
+}
+.blog{
+      background-color: rgba(255, 0, 0, 0.8);
+      font-size: 30px;
+      text-align: center;
+}
+.img5{
+  width: 190px;
+  height: 100px;
+}
+h5{
+  font-family: 'Poppins', sans-serif;
+  font-size: 17px;
+  color: #5a5a5a;
+}
+.t3{
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-size: 12px;
+  color: #303030;
+}
+.t4{
+  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+  font-size: 15px;
+}
+.blog-bar{
+  background-color: #04c2c9;
+  width: 50px;
+  height: 3px;
+}
+.blog-1{
+  margin-left: 150px;
+}
+.blog-2{
+  margin-left: 450px;
+  margin-top: -155px;
+}
+.blog-3{
+  margin-left: 740px;
+  margin-top: -155px;
+}
+.blog-4{
+  margin-left: 1040px;
+  margin-top: -155px;
+}
+
```

Comment:
Kindly make your site responsive as  60% of visitors that would access your websites would do you using their mobile phones.